lms/djangoapps/edraak_university/views.py

```diff
@@ -130,13 +183,31 @@ class UniversityIDUpdateView(CourseContextMixin, generic.UpdateView):
     template_name = 'edraak_university/instructor/update.html'
 
     # The email and full_name fields are written directly in the `update.html` file.
-    fields = ('university_id', 'section_number',)
+    fields = ('university_id',)
```

Comment:
They are built on model fields and cohort is not a model field. The result would be:
Unknown field(s) (cohort) specified for UniversityID as an error.